crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -181,6 +183,29 @@ impl Tracing {
             layers.push(console_output_layer.boxed());
         }
 
+        if self.file_log_settings.enabled {
+            let env_filter_layer = env_filter_builder(
+                self.file_log_settings.common_settings.environment_variable,
+                self.file_log_settings.default_level,
+            );
+
+            let file_appender = RollingFileAppender::builder()
+                .rotation(Rotation::HOURLY)
+                .filename_prefix(self.service_name.to_string())
+                .filename_suffix("tracing-rs.json")
+                .max_log_files(6)
+                .build(&self.file_log_settings.file_log_dir)
+                .expect("failed to initialize rolling file appender");
```

Comment:
suggestion: We should error here instead of panicking, because this function is fallible compared to before, see:

https://github.com/stackabletech/operator-rs/blob/29e920d13d03112c0000714cbda735b242585f5d/crates/stackable-operator/src/logging/mod.rs#L31